examples/compute-instance/main.tf

```
@@ -0,0 +1,82 @@
+# Providers
+# -> providers.tf
+
+# Customizable parameters
+locals {
+  my_zone     = "ch-gva-2"
+  my_template = "Linux Ubuntu 22.04 LTS 64-bit"
+}
+
+# Existing resources (<-> data sources)
+data "exoscale_compute_template" "my_template" {
+  zone = local.my_zone
+  name = local.my_template
+}
+
+data "exoscale_security_group" "default" {
+  name = "default"
+}
+
+# SSH
+# -> ssh.tf
+
+# Sample instance pool
+resource "exoscale_compute_instance" "my_big_instance" {
+  zone = local.my_zone
+  name = "my-big-instance"
+
+  template_id = data.exoscale_compute_template.my_template.id
+  type        = "standard.medium"
+  disk_size   = 10
+
+  ssh_key = exoscale_ssh_key.my_ssh_key.name
+
+  security_group_ids = [
+    data.exoscale_security_group.default.id,
+    exoscale_security_group.my_ssh_security_group.id,
+  ]
+}
+
+resource "exoscale_compute_instance" "my_small_instance" {
+  zone = local.my_zone
+  name = "my-small-instance"
+
+  template_id = data.exoscale_compute_template.my_template.id
+  type        = "standard.micro"
+  disk_size   = 10
+
+  ssh_key = exoscale_ssh_key.my_ssh_key.name
+
+  security_group_ids = [
+    data.exoscale_security_group.default.id,
+    exoscale_security_group.my_ssh_security_group.id,
+  ]
+
+  labels = {
+    testkey = "testvalue"
+  }
+}
+
+data "exoscale_compute_instance_list" "small_instances" {
+  zone = "ch-gva-2"
+
+  filter_regex {
```

```suggestion
  regex_match {
```

(maybe?)
Naming is always hard :smile: